src/server/notifications.js

```diff
@@ -16,6 +16,9 @@ async function getOrganizationOwner(organizationId) {
     .limit(1)
     .eqJoin('user_id', r.table('user'))('right')(0)
 }
+
+export const ASSIGNMENT_CREATED_SUBJECT = '[{organizationName}] New assignment: {campaignTitle}'
+export const ASSIGNMENT_CREATED_BODY = 'You just got a new texting assignment from {organizationName}. You can start sending texts right away: \n\n{todoUrl}'
```

Comment:
Wait, you can export a const that references another const variable that hasn't been declared yet?

Reply:
Do you mean e.g., `{organizationName}`? Note this isn't a template literal, it will be replaced below by a regex.